.gitignore

```diff
@@ -58,3 +58,7 @@ pytest/.consent
 # testnet genesis
 /near/res/testnet.json
 /near/res/testnet_genesis_records*.json
+
+# state migration home dirs
+/pytest/backward
```

Comment:
hm, given this folders are annoying indeed - may be worth putting it into /tmp/?

Reply:
Sounds good